From 5e704fe389ef56d89d14caf52ec45fb045670ece Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Thu, 14 Jul 2022 17:49:54 +0200
Subject: [PATCH 18/45] meson.build: fix build without stack-protector

Move -fstack-protector-strong from possible_cc_flags to
possible_link_flags to avoid a build failure on toolchains without ssp

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 4ed57a655..a145faf06 100644
--- a/meson.build
+++ b/meson.build
@@ -177,7 +177,6 @@ possible_cc_flags = [
     '-Wstrict-prototypes',
     '-fno-strict-aliasing',
     '-fstack-clash-protection',
-    '-fstack-protector-strong',
     '--param=ssp-buffer-size=4',
     '--mcet -fcf-protection',
     '-Werror=implicit-function-declaration',
@@ -215,6 +214,7 @@ possible_link_flags = [
     '-Wl,-z,now',
     '-Wl,-fuse-ld=gold',
     '-fstack-protector',
+    '-fstack-protector-strong',
 ]
 
 if sanitize == 'none'
-- 
2.37.2

